### DISCUSSION TITLE
Telegraf Bot Stopped Working after a while

### DISCUSSION THREAD

**Discussion by jeetendra-choudhary** [telegraf/telegraf#1245] [Q&A] [answered]
We are using Telegraf for Telegram, Bots stops working all of sudden and we have to restart after restart it starts working for a while and stops again. There is no specific pattern found so far and this happens at random time interval. The solution is hosted on AWS EC2 Server on ubuntu operating system. There has been no error on the logs as such.

Couldn't find any question on SO related to this most of it is either python implementation or not Telegraf implementation. Any pointer or direction would be greatly appreciated.

**Comment by wojpawlik** [comment, marked as answer]
Try enabling `handlerTimeout` in `Telegraf` options, see https://github.com/thedevs-network/the-guard-bot/blob/abffcb6790fc30b99c0867603c580e5a84d1f4cb/bot/index.js#L10-L11 and https://github.com/telegraf/telegraf/issues/735#issuecomment-736029465. This is being reworked for 4.0 release in #1247. Please mark this comment as an answer if it solves your problem.
